common/src/state/chats.rs

```diff
@@ -39,6 +39,9 @@ pub struct Chat {
     // only Some for group chats
     #[serde(default)]
     pub conversation_name: Option<String>,
+    // Only for group chats
+    #[serde(default)]
```

Comment:
State::Chat implements Serialize. `creator` needs to be added to that implementation. 
![image](https://user-images.githubusercontent.com/8636602/231552422-f4dbfeba-34ad-4f7f-aa9e-4171a612c55f.png)